specification/search/resource-manager/Microsoft.Search/stable/2020-03-13/search.json

```diff
@@ -449,7 +449,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/SearchService"
+              "$ref": "#/definitions/SearchService_Update"
```

Comment:
Change request body would be a big breaking change, which should not happen to stable. Any reason for it?

Reply:
I was told that the location property that I inherited from TrackedResource/Resource was required in PUT requests, but not required in PATCH requests.

Reply:
Em, I think in request body in PATCH, one only include those that can be modified (id, name, type, location is likely not changeable). E.g. storage account https://github.com/Azure/azure-rest-api-specs/blob/master/specification/storage/resource-manager/Microsoft.Storage/stable/2019-06-01/storage.json#L2794-L2833

Well, above is another issue. The question is whether this change breaks existing users.